src/main/java/util/csvparsing/CsvMonumentConverter.java

```diff
@@ -137,6 +142,91 @@ else if (dateArray.length == 3) {
         return result;
     }
 
+    /**
+     * Static method for converting a specified Monument into a CSV row
+     * @param monument - Monument to convert into a CSV row
+     * @return String - CSV row representing the specified Monument
+     */
+    public static String convertMonument(Monument monument) {
```

Comment:
Refactored this method to use some private helper methods. @benvogler I re-ran the GenerateMonumentDescriptionsApplication and everything seemed to match the original CSV generated, but double checking that the logic is still the same from the earlier commit would be appreciated.